src/chart/custom/CustomSeries.ts

```diff
@@ -319,7 +319,7 @@ export type CustomSeriesRenderItem = (
     api: CustomSeriesRenderItemAPI
 ) => CustomSeriesRenderItemReturn;
 
-export interface CustomSeriesOption extends
+export interface CustomSeriesOption<CallbackDataParams = never> extends
```

Comment:
Was this necessary? It seems to work without this line change and `= never` doesn't look a good way to define.

Reply:
I'm sorry for not getting back to you sooner. I had to provide `never` to minimize the needed change. If the "never" is omitted, it would require providing a generic argument to `CustomSeriesOption`, resulting in API changes.